contracts/cw20-ics20/src/ibc.rs

```diff
@@ -33,6 +33,9 @@ pub struct Ics20Packet {
     pub receiver: String,
     /// the sender address
     pub sender: String,
+    /// optional memo for the IBC transfer
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub memo: Option<String>,
```

Comment:
This serde decorator is redundant. This field is `Some(_)` if filled or skipped if empty by default.

Reply:
That makes sense. I just wanted to be extra sure, but I can remove it.

Reply:
Actually, I don't think that's right. Removing it would make https://github.com/CosmWasm/cw-plus/pull/868/files#diff-0e169309202ffd9411af7c77249aeba5cec5e78912f3560e8ef8bb5c3ad565dbR700 fail.

Reply:
Oh, I think you're right. I was thinking about deserialization, while during serialization serde transforms null into some value - None in this case.